packages/pds-ember/app/styles/pds/helpers/_typography.scss

```diff
@@ -0,0 +1,9 @@
+@use "../core/typography/config" as Typography;
+
+.pds--body-lg {
+  @include Typography.Body(L);
+}
+
+.pds--body-sm {
+  @include Typography.Body(S);
+}
```

Comment:
These seem out of place. Even though they're related to Typography, they're not core styles, but they rely on mixins from `core/typography`.  Do the class names make sense?